airflow/models/taskinstance.py

```diff
@@ -504,6 +791,34 @@ def _execute_callable(context: Context, **execute_callable_kwargs):
     return result
 
 
+def _set_ti_attrs(target, source):
+    # Fields ordered per model definition
+    target.start_date = source.start_date
+    target.end_date = source.end_date
+    target.duration = source.duration
+    target.state = source.state
+    target.try_number = source.try_number
+    target.max_tries = source.max_tries
+    target.hostname = source.hostname
+    target.unixname = source.unixname
+    target.job_id = source.job_id
+    target.pool = source.pool
+    target.pool_slots = source.pool_slots or 1
+    target.queue = source.queue
+    target.priority_weight = source.priority_weight
+    target.operator = source.operator
+    target.custom_operator_name = source.custom_operator_name
+    target.queued_dttm = source.queued_dttm
+    target.queued_by_job_id = source.queued_by_job_id
+    target.pid = source.pid
+    target.executor = source.executor
+    target.executor_config = source.executor_config
+    target.external_executor_id = source.external_executor_id
+    target.trigger_id = source.trigger_id
+    target.next_method = source.next_method
+    target.next_kwargs = source.next_kwargs
```

Comment:
I don't know if this is an inconsistency fro ma previous change but if I take a look to the fields in the model I am missing:
- updated_at
- rendered_map_index
- trigger_timeout

Reply:
thanks jens -- my goal was to not change the behavior -- i don't know why those fields were not included before.  but there could be a reason 🤷